dataplane/saiserver/routing.go

```diff
@@ -630,6 +631,8 @@ func (ri *routerInterface) CreateRouterInterface(ctx context.Context, req *saipb
 		return nil, err
 	}
 
+	// Here we associate a port to its output interface. How is output port assigned?
```

Comment:
when we create an interface we associate the interface to a port
we do this twice in two places: on to map in port to l3 interface and one on map to FIB output interface to output port

Reply:
Ack.